index.html

```diff
@@ -5,9 +5,51 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b89b7c8adf.js" crossorigin="anonymous"></script>
 </head>
 <body>
+
+  <main>
+
+    <nav>
+      <div class="logo"><h2>AS Pathak</h2></div> 
+      <img class="menu" src="media\Union.png" alt="hamburger" />
+    </nav>
+
```

Comment:
[OPTIONAL] this here should represent the end of the `header` tag. This is where you can include your `main` tag. The main will contain the different sections of your page until you get to the `footer`.